nvm.sh

```diff
@@ -1883,8 +1883,7 @@ nvm_install_source() {
     ./configure --prefix="${VERSION_PATH}" $ADDITIONAL_PARAMETERS && \
     $make -j "${NVM_MAKE_JOBS}" ${MAKE_CXX-} && \
     command rm -f "${VERSION_PATH}" 2>/dev/null && \
-    $make -j "${NVM_MAKE_JOBS}" ${MAKE_CXX-} install && \
-    command rm -rf "${TMPDIR}"
```

Comment:
That's never been a problem before - usually the error message is sufficient, and since the majority of users won't have the skills to debug, the ones that will also have the skills to go comment out the `rm` line (as they do now)